### DISCUSSION TITLE
Play area on Android phones seems too small

### DISCUSSION THREAD

**Discussion by NatureHog** [ppy/osu#15823] [Ideas] [answered]
I was testing out osu!lazer on my OnePlus Nord (6.44" 90Hz Amoled) and the default play area seems pretty small.

Would it be possible to increase the play area, depending on how small the screen is? (E.g. higher UI density = bigger play area?)

**Comment by bdach** [comment, marked as answer]
can you elaborate? preferably via a screenshot? i believe the play area it should be taking up the entire screen.

there is a UI scale slider if you think the UI elements are too small.

**Reply by NatureHog** [reply]
![Screenshot_20211126-192946.jpg](https://user-images.githubusercontent.com/55314658/143619884-b7f61972-64b4-40e7-8b91-228a19e6a39c.jpg)I hope this is sufficient.

**Reply by NatureHog** [reply]
![Screenshot_20211126-193219.jpg](https://user-images.githubusercontent.com/55314658/143620000-39ef3f17-84b7-444e-aec5-ad6bb5df06a0.jpg)These are the UI scale settings for the above screenshot.

**Reply by bdach** [reply]
fine tuning UI dimensions for mobile platforms has not been done yet and is a low priority in comparison to desktop. please look out for future changes in this area once the desktop client is in a more complete state.

**Reply by NatureHog** [reply]
Ok, thanks.
